webapp/components/interop-dashboard.js

```diff
@@ -630,6 +634,26 @@ class InteropDashboard extends PolymerElement {
     return this.getYearProp('focusAreas')[name][prop];
   }
 
+  // Add the stable or experimental label to a tests URL depending on the view.
+  formatTestsURL(testsURL, stable) {
+    // Don't try to add a label if the URL is undefined or empty.
+    if (!testsURL) {
+      return testsURL;
```

Comment:
```suggestion
      return '';
```
Let's return ''  so it does not return weird values that might  cause issues